libraries/helpers.rb

```diff
@@ -189,8 +189,8 @@ def secret
 module ClearSecretsBeforeSaving
   def save
     unless @chef_secret_attributes.nil?
-      @chef_secret_attributes.each do |attribute|
-        Chef::Log.info("Clearing #{attribute}")
+      @chef_secret_attributes.each_with_index do |attribute, i|
+        Chef::Log.info("Clearing #{attribute} and #{@chef_secret_attributes.size - 1} others") if i.zero?
```

Comment:
Maybe we can Log.info once the number of secrets, but Log.debug all attributes

Reply:
ok
```suggestion
        Chef::Log.info("Clearing #{attribute} and #{@chef_secret_attributes.size - 1} others") if i.zero?
        Chef::Log.debug("Clearing #{attribute}")
```